hack/util.sh

```diff
@@ -63,7 +63,7 @@ capz::util::should_build_ccm() {
 
 # all test regions must support AvailabilityZones
 capz::util::get_random_region() {
-    local REGIONS=("australiaeast" "canadacentral" "eastus" "eastus2" "northcentralus" "northeurope" "uksouth" "westeurope" "westus2")
+    local REGIONS=("australiaeast" "canadacentral" "germanywestcentral" "mexicocentral" "northcentralus" "northeurope" "uksouth" "switzerlandnorth" "brazilsouth")
```

Comment:
`germanywestcentral` and `francecentral` have increased quota and I think would be safe to try, but unless you were able to increase it (I wasn’t), we only have standard quota in `mexicocentral`, `brazilsouth`, and `switzerlandnorth` so we may bump up against that. 
Also could we keep the list alpha-sorted? I just find it easier to do navigate and to diff.

Reply:
what is "standard quota"?  I was able to bump the quotas for D2v3 and B2s past 150 for all them (most more than that)

Reply:
"Standard quota" or default appears to be 65. If you were able to raise it, then full steam ahead. I had tried increasing quota in those regions previously but been denied.